src/clients/express.ts

```diff
@@ -66,12 +68,15 @@ export const expressApp = async (apis: NetworkApis) => {
             // i know this is not a clean solution :(
             switch (network) {
                 case Network.astar:
+                    await astarApi.start();
```

Comment:
I feel like starting the API instance for every call is very heavy for the node. Any way to optimize this?

Reply:
This is the nauture of serverless. You have to code as if every request is server restart. In the start call i have added a flag _started. If it is already started it will return early. Thus starting the api once.

Reply:
![Screenshot from 2022-07-27 15-59-45](https://user-images.githubusercontent.com/874046/181266148-f2ef4e45-c7b7-4d75-8179-744d750369f5.png)